packages/components/organisms/f-offers/src/utils/helpers.js

```diff
@@ -0,0 +1,24 @@
+
+const setCookie = (name, value, minutes = 1) => {
+    let expires = '';
+    const millisecondsPerMinute = 60 * 1000;
+    if (minutes) {
+        const date = new Date();
+        date.setTime(date.getTime() + (minutes * millisecondsPerMinute));
+        expires = `; expires=${date.toUTCString()}`;
+    }
+    document.cookie = `${name}=${(value || '') + expires}; path=/`;
+};
+
+export const pushToDataLayer = (category, action, label) => {
+    window.trak.event({
```

Comment:
We probably also want `f-trak` added as a peer dependency if we're going to use this.

Reply:
yes, I found out what this is now, I am looking into the package.

Reply:
I have now included the f-trak library under peer dependency.